src/main/java/edu/wpi/first/wpilib/javainstaller/controllers/DeployController.java

```diff
@@ -163,12 +163,14 @@ private void setupJRE(Session roboRioSession) throws JSchException, IOException
         Platform.runLater(() -> commandLabel.setText("Setting permissions"));
         executeCommand(roboRioSession, "chmod +x " + JRE_INSTALL_LOCATION + "/JRE/bin/*", "Error when setting permissions on the executables");
         executeCommand(roboRioSession, "setcap 'cap_sys_nice=pe' " + JRE_INSTALL_LOCATION + "/JRE/bin/java", "Error setting priority capability");
+        executeCommand(roboRioSession, "echo '/usr/local/frc/JRE/lib/arm/jli' > /etc/ld.so.conf.d/java.conf", "Error setting ld.so.conf");
         m_logger.debug("Set permissions");
 
         // Delete the scp'd tar gz
         m_logger.debug("Cleaning up");
         Platform.runLater(() -> commandLabel.setText("Cleaning up installer files"));
         executeCommand(roboRioSession, "rm " + JRE_TGZ_LOCATION, "Could not clean up. The roboRIO MIGHT be ok to use, but you should submit the error logs anyway");
```

Comment:
We should log and display the reboot to the user. The command exits instantly, so we should update the above command label text to be "Cleaning up installer files and rebooting robot", and perhaps add an artificial second or two delay so the user can see it. We should also add another log line for the reboot execution.